provider/database_db_homes_data_source.go

```diff
@@ -25,28 +26,20 @@ func DBHomesDatasource() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
-			"limit": {
```

Comment:
Limit and page aren't very useful, but I don't think we can get rid of them. We should figure out a way to conditionally add these in the generator for existing resources and then I think we should mark them as deprecated.

Reply:
I would remove them because there is no valid use case for them